sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Rules/Utilities/UtilityAnalyzerBase.cs

```diff
@@ -44,6 +44,8 @@ public abstract class UtilityAnalyzerBase : SonarDiagnosticAnalyzer
 
         protected string WorkDirectoryBasePath { get; set; }
 
+        protected virtual bool AnalyzeAutogeneratedCode { get; set; }

```

Comment:
Naming consistency: why is this property called `Autogenerated` code rather than just `Generated`?